lib/nfsc.js

```diff
@@ -840,6 +840,29 @@ class V3 {
             return callback(null, path_or_obj_attrs, obj_attrs);
         });
     }
+
+    /**
+     * Procedure FSSTAT retrieves volatile file system state information.
+     *
+     * @param {root_fh}  root_fh file handle of the root inode of the export,
+     *                                returned by the mount procedure.
+     * @param {function} callback(err: null || {status: string,
+     *                                          obj_attrs: Object},
+     *                            obj_attrs: Object,
+     *                            tbytes, fbytes, abytes,
+     *                            tfiles, ffiles, afiles);
+     */
+    fsstat(root_fh, callback) {
+        this.client.fsstat3(root_fh,
```

Comment:
we are supposed to receive the invarsec?

Reply:
done